kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -9,22 +9,25 @@ ssui_root="$repos_root/manageiq-ui-self_service"
 
 rm -rf $app_root
 
-git clone <%= @appliance_checkout.remote %> $appliance_root
+git clone <%= "--separate-git-dir /tmp/appliance.git" if @build_type == "release" %> \
+        <%= @appliance_checkout.remote %> $appliance_root
```

Comment:
Many of us rely on the ability to git pull on the appliance's git repos or to checkout a git sha/tag/branch.  Will this be possible if the /tmp directory is gone after reboot?